torchvision/transforms/functional_tensor.py

```diff
@@ -72,6 +54,14 @@ def convert_image_dtype(image: torch.Tensor, dtype: torch.dtype = torch.float) -
     if image.dtype == dtype:
         return image
 
+    _max_values: Dict[torch.dtype, float] = {
+        torch.uint8: float(255),
+        torch.int8: float(127),
+        torch.int16: float(32767),
+        torch.int32: float(2147483647),
+        torch.int64: float(9223372036854775807),
```

Comment:
Should we have the unsigned versions for uint16, uint32 etc?

Reply:
I think there is no support for these types: https://pytorch.org/docs/stable/tensors.html#data-types